testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1511,10 +1511,14 @@
 end
 
 When(/^I (enable|disable) the necessary repositories before installing Prometheus exporters on this "([^"]*)"((?: without error control)?)$/) do |action, host, error_control|
-  common_repos = 'os_pool_repo os_update_repo tools_pool_repo tools_update_repo'
-  step %(I #{action} the repositories "#{common_repos}" on this "#{host}"#{error_control})
   node = get_target(host)
   _os_version, os_family = get_os_version(node)
+  common_repos = if os_family =~ /^opensuse/ || os_family =~ /^sles/
```

Comment:
I would say we can merge this if statement and the other one bellow...
But as you feel.

Reply:
Right (I did not understand your comment initially)